pkg/agent/qrm-plugins/util/consts.go

```diff
@@ -28,25 +28,30 @@ type AllocationHandler func(context.Context, *pluginapi.ResourceRequest) (*plugi
 type HintHandler func(context.Context, *pluginapi.ResourceRequest) (*pluginapi.ResourceHintsResponse, error)
 
 const (
-	MetricNameAllocateByCPUAdvisorServerCalled = "alloc_by_cpu_advisor_server_called"
-	MetricNameHandleAdvisorRespCalled          = "handle_advisor_resp_called"
-	MetricNameIsolatedPodNum                   = "isolated_pod_num"
-	MetricNamePoolSize                         = "pool_size"
-	MetricNameAllocateByCPUAdvisorServerFailed = "alloc_by_cpu_advisor_server_failed"
-	MetricNameHandleAdvisorRespFailed          = "handle_advisor_resp_failed"
-	MetricNameAllocateFailed                   = "alloc_failed"
-	MetricNameGetTopologyHintsFailed           = "get_topology_hints_failed"
-	MetricNameRemovePodFailed                  = "remove_pod_failed"
-	MetricNameLWCPUAdvisorServerFailed         = "lw_cpu_advisor_server_failed"
-	MetricNameLWMemoryAdvisorServerFailed      = "lw_memory_advisor_server_failed"
-	MetricNameHeartBeat                        = "heartbeat"
-	MetricNameRealStateInvalid                 = "real_state_invalid"
-	MetricNameCPUSetInvalid                    = "cpuset_invalid"
-	MetricNameCPUSetOverlap                    = "cpuset_overlap"
-	MetricNameMemSetInvalid                    = "memset_invalid"
-	MetricNameMemSetOverlap                    = "memset_overlap"
-	MetricNameNodeMemsetInvalid                = "node_memset_invalid"
-	MetricNameLWRecvStuck                      = "lw_recv_stuck"
+	MetricNameAllocateByCPUAdvisorServerCalled        = "alloc_by_cpu_advisor_server_called"
```

Comment:
can we re-arrange those const vars to several const-parts according to their scope? kind of mess since they become complicated now

Reply:
already classify them